e2e/test_operator_ha_upgrades/operator_ha_upgrade_test.go

```diff
@@ -168,7 +168,14 @@ var _ = Describe("Operator HA Upgrades", Label("e2e"), func() {
 		"upgrading a cluster with operator pod chaos and without foundationdb pod chaos",
 		func(beforeVersion string, targetVersion string) {
 			clusterSetup(beforeVersion, true /* = enableOperatorPodChaos */)
+			initialGeneration := fdbCluster.GetPrimary().GetStatus().Cluster.Generation
 			upgradeAndVerify(fdbCluster, targetVersion)
+			// Verify that the cluster generation number didn't increase by more
+			// than 40 (in an ideal case the number of recoveries that should happen
+			// during an upgrade is 9, but in reality that number could be higher
+			// because different server processes may get bounced at different times).
+			Expect(fdbCluster.GetPrimary().GetStatus().Cluster.Generation).To(BeNumerically("<=", initialGeneration+40))
```

Comment:
Could you create an issue for this? We should follow up on this, to understand why we see a higher number of generations than expected.

Reply:
Ok.

Reply:
Created issue: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/1607